assembl/static2/js/app/components/common/footer.jsx

```diff
@@ -76,46 +81,14 @@ const Footer = ({
                 bluenove
               </Link>
             </div>
-            <div className="terms">
-              {hasTermsAndConditions && (
-                <div className="terms-of-use">
-                  <Link to={`${get('terms', debateSlug)}`}>
-                    <Translate value="footer.terms" />
-                  </Link>
-                </div>
-              )}
-              {hasLegalNotice && (
-                <div className="legal-notice">
-                  {hasTermsAndConditions ? <span className="small-hyphen-padding"> &mdash; </span> : null}
-                  <Link to={`${get('legalNotice', debateSlug)}`}>
-                    <Translate value="footer.legalNotice" />
-                  </Link>
-                </div>
-              )}
-            </div>
-            <div className="terms">
-              {hasCookiesPolicy && (
-                <div className="cookie-policy">
-                  <Link to={`${get('cookiesPolicy', debateSlug)}`}>
-                    <Translate value="footer.cookiePolicy" />
-                  </Link>
-                </div>
-              )}
-              {hasPrivacyPolicy && (
-                <div className="privacy-policy">
-                  {hasCookiesPolicy ? <span className="small-hyphen-padding"> &mdash; </span> : null}
-                  <Link to={`${get('privacyPolicy', debateSlug)}`}>
-                    <Translate value="footer.privacyPolicy" />
-                  </Link>
-                </div>
-              )}
-              {hasUserGuidelines && (
-                <div className="user-guidelines">
-                  {hasPrivacyPolicy || hasCookiesPolicy ? <span className="small-hyphen-padding"> &mdash; </span> : null}
-                  <Link to={`${get('userGuidelines', debateSlug)}`}>
-                    <Translate value="footer.userGuidelines" />
-                  </Link>
-                </div>
+            <div className="legal-pages">
+              {Object.keys(legalPages).map(
```

Comment:
👏 great! Just for a cleaner JSX I'd have this in a renderFooterLinks function and simply do {renderFooterLinks()} here

Reply:
the cookie page needs to be always here because of the cookie configuration section